experimental_official_dockerfiles/master/devel.packages.txt

```diff
@@ -0,0 +1,54 @@
+# All required CUDA packages
```

Comment:
I will separate here GPU specific packages in a file and the common CPU and GPU dependencies in another file.  This will minimize version misalignment if we manage the same dependencies in two files when we will produce the devel CPU image.

Reply:
I think two concrete and complete files would be easier to maintain than three partial files. Our list is not very long.

Reply:
Hmm, hard to say, now that I think about it. I'd like to see how it would look, if possible.

Reply:
Oh, right. We don't need a devel CPU image.

Reply:
> We don't need a devel CPU image.
We don't need it?

Reply:
No, I don't think so. That would more extra maintenance to the Dockerfile than I'd like. GPU can build both the CPU and GPU packages.

Reply:
Technically I know but I don't think that we will have too much overhead if we just separate on a file the common CPU/GPU dependencies and the GPU only dependencies in another one. I think that the image download size diff between GPU and CPU devel it will be quite important to just distribute a single image.

Reply:
Yeah, it depends on overhead. As long as it can be really simple, then it might be worth it. Maybe once we get this base version merged you could try it in a PR?

Reply:
>  Maybe once we get this base version merged you could try it in a PR?
Yes as you want I think you can isolate a `devel.packages_gpu_only.txt` now or later.

Reply:
Now on Dockehub we have compressed size:
`devel` 1.21 GB
`devel-gpu` GPU 3.56 GB